### DISCUSSION TITLE
Build Log shows /_not-found

### DISCUSSION THREAD

**Discussion by chris-nowicki** [vercel/next.js#57839] [Help] [answered]
### Summary

My build log shows /_not-found and I am not sure how to figure out what is causing it.  It is driving me nuts!! Any help / idea are appreciated :)

My site runs perfectly fine and no information seems to be missing.

![Screenshot 2023-10-31 at 5 35 32 PM](https://github.com/vercel/next.js/assets/102450568/c9b2ed4a-97b3-4f03-aaa8-682241328644)

repo: https://github.com/chris-nowicki/chrisnowicki.io

**Comment by icyJoseph** [comment, marked as answer]
Hi,

Why did that worry you? It is the page that Next.js will serve for 404'd paths.

You can also make a custom `app/not-found.tsx` file, if you want to customize how that 404 page looks.

**Reply by chris-nowicki** [reply]
Thanks @icyJoseph ... I didn't realize that.  Appreciate it 👍🏻
